### PR TITLE
full-code

### PR DESCRIPTION
**Pull request by Alzubair98**
the full code is ready now

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+        
+      </div>
+         
+        
+      
+    </div>
+    
+
+    <!-- <div><p><strong>Alzubair</strong></p></div> -->
+    <!-- <button>click me</button> -->
```
Comment (Alzubair98) [inline index.html:210]: leaving commented code will not make your code clean 🗑️ . please remove it

File: README.md [hunk 2]
```diff
@@ -1,2 +1,97 @@
...
+## Have you already completed the Sass lessons?
+
+If you completed the Sass lessons you may prefer to work with SCSS files, if that's the case you need to perform
+a few modifications to the linter configuration files.
+
+1. In the file `.github/workflows/linters.yml` [line 48](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml#L48) replace `"**/*.{css,scss}"` with `"**/*.scss"`
+2. Use `npx stylelint "**/*.scss"` to lint you SCSS files and not the generated CSS
+
+Once you've done the previous steps you can focus on your SCSS files and not in fixing the generated CSS output
+every time.
```
Comment (Alzubair98) [inline README.md:97]: please change your readme file. follow the template and make it the same as the template.

the template for the readme file to be used is as follows, [follow this link](https://github.com/microverseinc/readme-template/blob/master/README.md).

 Copy the template by clicking the two boxes in the image below once you go to the link above.
image
image


 Then clear your readme file and paste the template in your readme file on your machine.

 Lastly edit the readme file by adding project name, git hub link and LinkedIn link, as well as the issues link for your current project.

**Review by Alzubair98 — COMMENTED** [review]
you are almost done 💯💯
 
 you have passed:
 1- linters
 2- good pull request description 
 3- good push description
 
 #changes required!!
 
 please follow the instruction to complete the changes

**Review by Chimwemwe-127001 — APPROVED** [review]
# Status: Approved ✔️ ✔️ ✔️ 💯 
### Hi @Alzubair98hi

You've done an amazing job with the code review :fire: :tada: 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

## Highlights :+1: 
:heavy_check_mark: Good utilization of GitHub inline comments feature in the review
:heavy_check_mark: Great job spotting the lack of semantic tags

## A few things to always keep in mind ♻️
- Your comments should always be phrased to sound more suggestive.
- Always include an explanation, a "why", for each requested change 
- Make sure that your comments are clear and easy to understand
- Use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖

Check the comments under the review to find which comments can be altered.

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**